README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
```

Comment:
Consider adding error handling for the installation and execution commands to ensure users are notified if an installation fails or if the server does not start as expected.